validator/src/main.rs

```diff
@@ -2119,6 +2126,15 @@ pub fn main() {
                     .default_value("10")
                     .help("Minimum time that the validator should not be leader before restarting")
             )
+            .arg(
+                Arg::with_name("max_delinquent_stake")
+                    .long("max-delinquent-stake")
+                    .takes_value(true)
+                    .validator(solana_clap_utils:: input_validators::is_valid_percentage)
```

Comment:
Sorry, just throw `is_valid_percentage` in the global `use` at the top of the file and then this and the other `.validator()` reduce down to:
```suggestion
                    .validator(is_valid_percentage)
```